contracts/sol6/IKyberHint.sol

```diff
@@ -4,9 +4,10 @@ import "./IKyberReserve.sol";
 
 
 interface IKyberHint {
-    enum TradeType {MaskIn, MaskOut, Split}
+    enum TradeType {BestOfAll, MaskIn, MaskOut, Split}
     enum HintErrors {
         NoError, // Hint is valid
+        BestOfAllHintError, // reserveIDs and splits must be empty for BestOfAll hint
```

Comment:
seems like its inconsistent to add hint as part of the name.

its not done in any other error

Reply:
Change to `NoneEmptyDataError`.